### ISSUE TITLE
Cannot Find Message Text Field with VoiceOver

### ISSUE THREAD

**Issue by Simon818** [TelegramMessenger/Telegram-iOS#1155]
### Checklist
- ☑ I am reporting an issue in existing functionality that does not work as intended
- ☑ I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Describe the issue that you are experiencing

VoiceOver users cannot manually locate the message text field.

### Expected Behavior
Tell us what should happen

Touching the message text field should cause VoiceOver to read it, also allowing the user to double-tap and bring up the keyboard.

### Actual Behavior
Tell us what happens instead

VoiceOver makes a sound to indicate that there is no item at that location. It is possible to navigate to the text field manually by finding the voice message button and swiping right a few times, but its location changes when the keyboard is visible and trying to swipe past the last visible message to omve further down the screen does not work.

### Steps to Reproduce
1. Open a conversation
2. Load VoiceOver
3. Try to touch the text field and note that VoiceOver makes a clunking sound to indicate it cannot detect anything at this location on the screen.

### Environment
**Device:** iPhone 13

**iOS version**: 16.6

**App version:** 9.6.5

**Comment by AeroSmyte** [comment]
Does this seem like something a beginner could hop in on and tackle? I would love to help contribute if I can!
